edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -13,27 +15,39 @@
     jwt_cookie_signature_name,
 )
 from edx_rest_framework_extensions.auth.jwt.constants import JWT_DELIMITER
-from edx_rest_framework_extensions.permissions import NotJwtRestrictedApplication
+from edx_rest_framework_extensions.permissions import LoginRequired, NotJwtRestrictedApplication
 
 log = logging.getLogger(__name__)
 USE_JWT_COOKIE_HEADER = 'HTTP_USE_JWT_COOKIE'
 
 
+def _includes_base_class(iter_classes, base_class):
```

Comment:
Nit: I've typically been following the top-down rule from the `Clean Code` book for readability.  So that the helper functions/methods are below the higher-level public methods.

Reply:
I'll see if it is simple enough.  I think I prefer what I did out of habit, not because it is better. :)

Reply:
Done.